models/LM-tensorflow/test_LM_tensorflow.py

```diff
@@ -75,13 +75,13 @@ def side_effect_cache(args):
 		args.cache = True
 		main(args)
 	mock = mocker.patch('main.main', side_effect=side_effect_train)
-	run()
+	run([])
```

Comment:
maybe you can use run([XXX, XX, XXX, XXX]) instead of the side_effect for each model ?